p_sets/p_set_1.rb

```diff
@@ -51,5 +51,41 @@ def mario
   pyramid_printer calculate_row_length ask_height
 end
 
+#TIMEFORCHANGE
+#assume that your program’s input will be 9.00 or just 9 but, again, not $9 or 900.
+
+COINS = [100,20,10,5,2,1]
+
+def ask_change
+  while true
+    puts "How much change is owed?"
+    change = gets.chomp.to_f
+    break if change > 0
+  end
+   return change
+end
+
+def calculate_coins(pence)
+  n = COINS.count
+  coins = []
+  for n in 0..n-1
+    denomination = COINS[n]
+    quantity = pence / denomination
+    pence -= (denomination * quantity)
+    coins << quantity
+  end
+   coins.inject(:+)
+end
+
+def convert_to_pence(change)
+  (change * 100).round
+end
+
+def change_reporter
+  puts (calculate_coins convert_to_pence ask_change).to_s
```

Comment:
Yep, and additionally if the argument list to a function is more than just a simple list of values, I'd probably use parentheses for readability. 
```
calculate_coins convert_to_pence ask_change # bad

calculate_coins(convert_to_pence(ask_change)) # good
```
There are quickly cases where the order or arguments gets confusing in this way:
```
calculate_coins convert_to_pence ask_change, true # which method gets passed `true`?
```